Rohit/PostgresAssignment/controller/class.ts

```diff
@@ -0,0 +1,53 @@
+import { Context } from "vm";
+
+const cls = require('../sql/class');
+
+interface classesinfo {
+    cls_id: number;
+    st_id: number;
+  }
+  
+  
+exports.classData = async (ctx: Context) => {
+    try {
+      let [rows]: Array<{ rows: classesinfo }> = [];
+      rows = await cls.get_classes();
```

Comment:
can we do it in 1 line
      let [rows]: Array<{ rows: classesinfo }> = await cls.get_classes();